jinahub/rankers/MinRanker/manifest.yml

```diff
@@ -0,0 +1,8 @@
+manifest_version: 1
+name: MinRanker
```

Comment:
make sure you push to hubble and add the keys to secrets

Reply:
yes, https://github.com/jina-ai/executors-secrets/pull/3